src/modules/endpoint/DatabaseConnector.js

```diff
@@ -187,6 +201,40 @@ class DatabaseConnector {
     await this.writeFile(path, JSON.stringify(languages))
   }
 
+  async storeCities (cities: Array<CityModel>) {
+    const jsonModels = cities.map((city: CityModel): ContentCityJsonType => ({
+      name: city.name,
+      live: city.live,
+      code: city.code,
+      extrasEnabled: city.extrasEnabled,
+      eventsEnabled: city.eventsEnabled,
+      sortingName: city.sortingName
+    }))
+
+    await this.writeFile(this.getContentPath('cities'), JSON.stringify((jsonModels)))
```

Comment:
```suggestion
    await this.writeFile(this.getContentPath('cities'), JSON.stringify(jsonModels))
```
one pair of brackets unnecessary :)